hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/ReplicationSupervisor.java

```diff
@@ -232,14 +232,38 @@ private ReplicationSupervisor(StateContext context, ExecutorService executor,
    * Queue an asynchronous download of the given container.
    */
   public void addTask(AbstractReplicationTask task) {
+    if (queueHasRoomFor(task)){
```

Comment:
[checkstyle](https://github.com/len548/ozone/actions/runs/13049848933/job/36407507462#step:8:16) complains:

```
hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/ReplicationSupervisor.java
 235: '{' is not preceded with whitespace.
```

```suggestion
    if (queueHasRoomFor(task)) {
```